### PR TITLE
registering,login, all processes connected with users stuff

### PR DESCRIPTION
**Pull request by szmo**
https://trello.com/c/CrhmUmmt/20-dodanie-kont-u%C5%BCytkownik%C3%B3w

### CODE REVIEW

**Comment by evemorgen** [comment]
There should be ```sign up``` button next to ```log in``` button
screen shot 2017-11-13 at 12 18 27
-- edit
done in c1da721

**Comment by evemorgen** [comment]
Kodus for being ```rwd``` out of the box :D
screen shot 2017-11-13 at 12 22 12

**Comment by evemorgen** [comment]
```sign up``` button overlays ```confirm password``` text box, I guess there should be ```margin-top``` or sth similar
screen shot 2017-11-13 at 12 23 27

File: rails_app/db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20171111205747) do
+
+  create_table "users", force: :cascade do |t|
+    t.string "login", default: "", null: false
```
Comment (evemorgen) [inline rails_app/db/schema.rb:16]: Not sure if default value of empty string is suitable for login, lol. Besides that defining default value for ```not null``` is pointless. Same applies to rest of stuff defined in similar fashion.

File: rails_app/config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,277 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
```
Comment (evemorgen) [inline rails_app/config/initializers/devise.rb:1]: This file is totally unreadable. Comments are over 93% of this file (sic!)

File: rails_app/app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (evemorgen) [inline rails_app/app/models/user.rb:4]: maybe it is worth considering to add ```Timeoutable```

File: rails_app/app/views/layouts/_header.html.haml [hunk 4]
```diff
@@ -10,9 +10,22 @@
         = link_to 'Price monitor', root_url, class: 'navbar-brand'
```
Comment (evemorgen) [inline rails_app/app/views/layouts/_header.html.haml:11]: I guess we should add here our cool new logo:
screen shot 2017-11-13 at 13 38 22

**Review by evemorgen — CHANGES_REQUESTED** [review]

File: rails_app/config/initializers/devise.rb (hunk 2 above)
Comment (szmo) [reply rails_app/config/initializers/devise.rb:1]: i would rather left is in this state

Comment (evemorgen) [reply rails_app/config/initializers/devise.rb:1]: ok

File: rails_app/app/models/user.rb (hunk 3 above)
Comment (evemorgen) [reply rails_app/app/models/user.rb:4]: not worth spending time on this now

**Review by evemorgen — APPROVED** [review]

**Comment by evemorgen** [comment]
ok

**Pull request merged**